### PR TITLE
Added Django docker multistage cache example project

### PR DESCRIPTION
**Pull request by Kandeel4411**
## [Docker multistage example]( https://github.com/c-hive/basics/issues/10)
Hello! I am not familiar with Rails so I made this example using Django which should basically be of the same procedure to leverage the intermediate layer caches. This could also serve as  the basis of pushing/pulling built intermediate images using CI pipelines to leverage their cache as well.

Let me know if this is what you had in mind!

### CODE REVIEW

**Review by thisismydesign — COMMENTED** [review]
This is great, thanks! Could you add a link to the main readme?

**Comment by Kandeel4411** [comment]
Oh right! Got it

**Review by thisismydesign — APPROVED** [review]

**Comment by thisismydesign** [comment]
Thanks again!

**Comment by Kandeel4411** [comment]
My pleasure :)!

File: docker-multistage-cache/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+- Build the intermediate dependency stages into separate images with `--target` being the stage's name and a `-t` tag that you could later identify it with. I.e:
+
+  ```bash
+  docker build --target=dependencies -t django_example:dep .
+  ```
+
+- Build the final image use `--cache-from` listing the intermediate image. I.e:
+
+  ```bash
+  docker build --cache-from django_example:dev -t django_example:final .
```
Comment (max-sixty) [inline docker-multistage-cache/README.md:20]: Should this be `django_example:dep`?

Comment (Kandeel4411) [reply docker-multistage-cache/README.md:20]: You are right, it must have slipped through.

Comment (max-sixty) [reply docker-multistage-cache/README.md:20]: Cheers!

**Pull request merged**